linker error `status.c:(.text+0x31c): undefined reference to `curl_easy_strerror'`

iamb errors when linking with error 
```
  = note: /usr/bin/ld: /home/gireesh/gits/iamb/target/debug/deps/libsixel_sys_static-9bc14cb98e1499fc.rlib(libsixel_la-status.o): in function `sixel_helper_format_error':
          status.c:(.text+0x31c): undefined reference to `curl_easy_strerror'
          collect2: error: ld returned 1 exit status

  = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)

warning: `iamb` (bin "iamb") generated 1 warning
error: could not compile `iamb` (bin "iamb") due to previous error; 1 warning emitted
```

after bisecting found that it is in this 221faa828df1842abb1680b572cfb39b85289331 commit
by disabling `sixel` feature compiled it successfully

platform: Archlinux

libcurl is already present

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -44,9 +44,9 @@ jobs:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         reporter: 'github-check'
     - name: Run tests
-      run: cargo test ${{ matrix.platform == 'windows-latest' && '--no-default-features' || '' }}
+      run: cargo test
     - name: Build artifacts
-      run: cargo build --release ${{ matrix.platform == 'windows-latest' && '--no-default-features' || '' }}
+      run: cargo build --release
     - name: Upload artifacts
       uses: actions/upload-artifact@master
       with:

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,7 +40,7 @@ markup5ever_rcdom = "0.2.0"
 mime = "^0.3.16"
 mime_guess = "^2.0.4"
 open = "3.2.0"
-ratatui-image = { version = "0.4.1", features = ["serde"] }
+ratatui-image = { version = "0.4.3", features = ["serde"] }
 regex = "^1.5"
 rpassword = "^7.2"
 serde = "^1.0"

